### PR TITLE
Fix var with underscore issue

### PR DESCRIPTION
**Pull request by djkevincr**
Added validation for the following.
var _ = 100;
int _ = 100;

### CODE REVIEW

File: tests/ballerina-test/src/test/resources/test-src/types/var/var-ignore-negative.bal [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function testVarNegativeCases() returns (int) {
+    int _ = 100;
+    var _ = 200;
+    return 0;
+}
```
Comment (anupama-pathirage) [inline tests/ballerina-test/src/test/resources/test-src/types/var/var-ignore-negative.bal]: Shall we add new line here?

Comment (hasithaa) [reply tests/ballerina-test/src/test/resources/test-src/types/var/var-ignore-negative.bal]: +1

Comment (djkevincr) [reply tests/ballerina-test/src/test/resources/test-src/types/var/var-ignore-negative.bal]: +1

**Comment by hasithaa** [comment]
@djkevincr , some of the JMS test cases are failing. Can you check. ?

Failed tests: 
  JMSConnectorTestCase.testDurableTopicSubscriberPublisher:89 Â» BallerinaTest Ti...
  JMSConnectorTestCase.testJMSMapMessagePublisherAndSubscriber:173 Â» BallerinaTest
  JMSConnectorTestCase.testJMSProperties:120 Â» BallerinaTest Unable to start car...
  JMSConnectorTestCase.testJmsSimpleQueueReceiverProducer:100 Â» BallerinaTest Un...
  JMSConnectorTestCase.testMbSimpleQueueReceiverProducer:138 Â» BallerinaTest Una...
  JMSConnectorTestCase.testMbSimpleTopicSubscriberPublisher:156 Â» BallerinaTest ...
  JMSConnectorTestCase.testQueueConsumerProducer:47 Â» BallerinaTest Unable to st...
  JMSConnectorTestCase.testTopicSubscriberPublisher:64 Â» BallerinaTest Unable to...
  MetricsTestCase.setup:64 Â» BallerinaTest Unable to start carbon server on port...
  TracingTestCase.setup:57 Â» BallerinaTest Unable to start carbon server on port...
  UnaryBlockingBasicTestCase.setup:58 Â» BallerinaTest Unable to start carbon ser...
  UnaryBlockingHeaderTestCase.setup:47 Â» BallerinaTest Unable to start carbon se...
  StreamsWithinServicesTestCase.testStreamsWithinServices:51->startServer:86 Â» BallerinaTest

Tests run: 184, Failures: 13, Errors: 0, Skipped: 51

**Comment by djkevincr** [comment]
@hasithaa +1 updated the PR.

**Pull request merged**